src/main/java/reactor/netty/http/client/HttpClientOperations.java

```diff
@@ -16,9 +16,7 @@
 
 package reactor.netty.http.client;
 
-import java.net.InetSocketAddress;
-import java.net.URI;
-import java.net.URISyntaxException;
+import java.net.*;
```

Comment:
do not use `.*` for the imports

Reply:
My IDE is trying to be helpful, but in this case it was not. I fixed it!